packages/angular/cli/commands/add-impl.ts

```diff
@@ -139,6 +148,29 @@ export class AddCommand extends SchematicCommand<AddCommandSchema> {
     return this.executeSchematic(collectionName, options['--']);
   }
 
+  async reportAnalytics(
+    paths: string[],
+    options: AddCommandSchema & Arguments,
+    dimensions: (boolean | number | string)[] = [],
+    metrics: (boolean | number | string)[] = [],
+  ): Promise<void> {
+    const collection = options.collection;
+    function _matches(re: RegExp | string, str: string): boolean {
+      if (typeof re == 'string') {
+        return str === re;
+      } else {
+        return re.test(str);
+      }
+    }
+
+    // Add the collection if it's safe listed.
+    if (collection && safelistNgAddCollectionAnalytics.find(re => _matches(re, collection))) {
```

Comment:
nit: `.find` -> `.some`

Reply:
Using a call to `isPackageNameSafeForAnalytics(collection)` instead.